style/main.css

```diff
@@ -104,55 +93,26 @@ body {
     background-color:#B62F2F;
     border: 1px solid #B62F2F;
     outline: none;
-
 }
 
-/*.button_o {*/
-/*    background-color: transparent;*/
-/*    border: 1px solid #232020;*/
-/*}*/
-
 .button:hover {
     background-color: #990a0a;
     border-color: #990a0a;
 }
 
-
-
-/*.button_o:focus-visible {*/
-/*    border-color: #FFC700;*/
-/*}*/
-
 .button:active{
     transform: translate(2px, 2px);
 }
-/*.button:disabled{*/
-/*    background-color: #E0E0E0;*/
-/*    border-color: #E0E0E0;*/
-/*    color:#898989;*/
-/*    cursor:default;*/
-/*    transform: none;*/
-/*}*/
 
 .section-offset {
     padding: 40px 80px;
-
 }
 
-
-/*.gray-block {*/
-/*    background-color: #F8F8F8;*/
-/*    border-radius: 12px;*/
-/*    padding: 50px;*/
-/*    min-height: 530px;*/
-/*}*/
-@media (max-width: 991px) {}
 @media (max-width: 767px) {
```

Comment:
Use for mobile — 320 px, 480 px
for tablet — 768 px
for netbook — 1024 px 
for monitors— 1280 px, 1600 px, 1920 px